src/upload-tests.proj

```diff
@@ -25,6 +25,9 @@
          - Command must produce testResults.xml on successful execution.
          - Command is expressed in terms of an .SH file if on *Nix -->
     <UseScriptRunner>true</UseScriptRunner>
+    <!-- For Windows ARM & ARM64 UAPAOT runs, we'll need a special runner that knows how to do this (TODO: Combine the runners) -->
+    <UseContinuationRunner Condition="'$(ArchGroup)'=='arm' AND '$(TargetGroup)'=='uapaot'">true</UseContinuationRunner>
+    <SecondaryPayloadDir   Condition="'$(ArchGroup)'=='arm' AND '$(TargetGroup)'=='uapaot'">%HELIX_WORKITEM_PAYLOAD%\native</SecondaryPayloadDir>
```

Comment:
Are these two properties going to be used only on buildtools targets? If so, should we just set them there to keep this file clean?